50SAT/SPEX$50SAT.tex

```diff
@@ -0,0 +1,353 @@
+If this project is conducted over a semester timeline, the team will need to expediently cover the required areas defined below.
+
+This project will take place over three technical domains: electrical, mechanical, and software.
+
+For the electrical systems, the team will need to review the existing PCB designs, primarily to determine whether it is feasible to manufacture and populate them on campus.
+Also the team will need to determine if all components required are still available, and if not, find suitable replacements and make the necessary design changes.
+PCB assembly will be the system with the longest lead time and should be prioritized first to ensure no bottlenecks on the critical path.
+Once assembled, each will need to be tested, and integrated within the system structure.
+
+For the mechanical systems, the team should evaluate the feasibility of manufacturing the frame on-campus.
+Any tweaks to the internal structure and placement of internal components should be locked down as early as possible.
+The mechanical team will also be responsible for testing the frame and whole assembly for verification against thermal and vibration requirements.
+
+For the onboard software, the team should review the existing code, and try to demonstrate functionality on similar hardware to prevent dependency on the electrical team.
+Once the primary electronics are ready, the software should be tested for functionality on the system hardware.
+
+The timeline below is preliminary:
+
+\begin{table}
+\centering
+ \begin{tabular}{|c | c | c|}
+ \hline
+ Phase & Task & Duration \\ [0.5ex]
+ \hline\hline
+ 1 & Review the existing \$50SAT designs and materials &  no more than 2 weeks \\
+ \hline
+ 2 & Subsystem development & 6 weeks \\
+ \hline
+ 2 & Order PCB design and/or assembly & 6 weeks \\
+ \hline
+ 2 & Review changes to mechanical strcuture and order materials & no more than 2 weeks \\
+ \hline
+ 2 & Testing of individual subsystems & 2 weeks \\
+ \hline
+ 3 & System Assembly & 1 week  \\
+ \hline
+ 4 & System testing & 2 weeks  \\
+ \hline
+ 5 & Generate documentation and delivery to SPEX & 1 week  \\
+ \hline
+\end{tabular}
+\label{tab:timeline}
+\end{table}
+
+\begin{itemize}
+  \item Review the existing \$50SAT designs and materials - no more than 2 weeks
+  \item Subsystem development - concurrent
+  \begin{enumerate}
+    \item Order PCB design and/or assembly - 6 weeks
+    \item Review changes to mechanical strcuture and order materials - no more than 2 weeks
+    \item Setup software environment and prototyping test bed - no more than 4 weeks
+  \end{enumerate}\
+  \item Testing of individual subsystems - 2 weeks
+  \item System Assembly - 1 week
+  \item System testing - 2 weeks
+  \item Generate documentation and delivery to SPEX
+\end{itemize}
+
+Schedule slip is tolerable, since there are few external considerations within the fall semester, e.g. a competition or public display event.
```

Comment:
Add interword space: `e.g.\`

See this explanation: https://tex.stackexchange.com/questions/2734/taking-unncessary-space-after-e-g-or-i-e